lib/chewy/journal/apply.rb

```diff
@@ -1,6 +1,8 @@
 module Chewy
   class Journal
     module Apply
+    module_function
```

Comment:
The indentation is correct, just as I like it. Going to refactor it further definitely.